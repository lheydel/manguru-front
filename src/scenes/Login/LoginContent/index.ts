import { LoginContent } from './LoginContent';
export default LoginContent;
export * from './LoginContent';
