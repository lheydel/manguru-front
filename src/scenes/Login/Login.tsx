import React from 'react';
import LoginContent from './LoginContent';

export const Login: React.FC = () => (
  <LoginContent />
);
