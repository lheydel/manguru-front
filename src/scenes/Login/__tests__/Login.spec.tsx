import React from 'react';
import Login from '..';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Login component', () => {

  it('should render itself', () => {
    const wrapper = shallow(<Login />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
