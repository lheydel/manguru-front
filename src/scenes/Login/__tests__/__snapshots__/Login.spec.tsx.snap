// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component should render itself 1`] = `<Connect(RawLoginContent) />`;
